keras_nlp/models/bert/bert_classifier_test.py

```diff
@@ -26,17 +26,18 @@ def setUp(self):
         # Setup model.
         self.vocab = ["[PAD]", "[UNK]", "[CLS]", "[SEP]", "[MASK]"]
         self.vocab += ["the", "quick", "brown", "fox", "."]
+        self.sequence_length = 5
         self.preprocessor = BertPreprocessor(
             BertTokenizer(vocabulary=self.vocab),
-            sequence_length=5,
+            sequence_length=self.sequence_length,
         )
         self.backbone = BertBackbone(
             vocabulary_size=self.preprocessor.tokenizer.vocabulary_size(),
             num_layers=2,
             num_heads=2,
             hidden_dim=2,
             intermediate_dim=4,
-            max_sequence_length=self.preprocessor.packer.sequence_length,
+            max_sequence_length=self.sequence_length,
```

Comment:
We could just do `self.preprocessor.sequence_length` here to avoid attaching more state to the test case.

Reply:
Fixed.